jme3-testdata/src/main/resources/Scenes/LightProbes/license.txt

```diff
@@ -0,0 +1,16 @@
+Licensing history for files in "src/main/resources/Textures/Sky/quarry_03"
+
+Based on "Quarry 03":
+* download file:     quarry_03.jpg
+* downloaded from:   https://hdrihaven.com/hdri/?h=quarry_03
+* author:            Sergej Majboroda
+* author URL:        https://hdrihaven.com/hdris/?a=Sergej%20Majboroda
+* license type:      CC0
+* license URL:       https://creativecommons.org/publicdomain/zero/1.0/
+
+Derived files (also licensed CC0):
+* equirec_4k.jpg
+* probe.j3o
```

Comment:
filename looks incorrect

Reply:
I've removed "equirec_4k.jpg" from the list since I did not use it, and updated probe.j3o to match name I gave it of "quarry_Probe.j3o"